src/routes/manage-selections/__tests__/ManageSelectionsScreen.spec.tsx

```diff
@@ -6,7 +7,7 @@ import labels from '../../../constants/labels.json'
 import { useLoadDiscipline } from '../../../hooks/useLoadDiscipline'
 import useReadCustomDisciplines from '../../../hooks/useReadCustomDisciplines'
 import useReadSelectedProfessions from '../../../hooks/useReadSelectedProfessions'
-import AsyncStorage from '../../../services/AsyncStorage'
+import AsyncStorageService from '../../../services/AsyncStorage'
```

Comment:
I think now we mixed up naming. Sometimes we have AsyncStoargeService (for local) and AsyncStorage (for RN)
and sometime we have
AsnycStorage (for local) and RNAsyncStorage (for RN)
I have no strong preference which one is better, but we should always use the same.